### DISCUSSION TITLE
Unable to cancel my subscription for a deleted account

### DISCUSSION THREAD

**Discussion by ghost** [community/community#35177] [Copilot Conversations] [answered]
I deleted a GitHub account and I am still debited for my Copilot subscription with no way to cancel it (you need access to your GitHub account to manage your subscription). Any idea welcome.

**Comment by radar-blip** [comment]
Related post here, maybe it can be of help: #34299

**Reply by ghost** [reply]
Thank you @linuxlawson but the issue here is that the account has been deleted.

**Comment by iamcjwilliams** [comment]
@jbrudant Visit here to [cancel your subscription](https://docs.github.com/en/billing/managing-billing-for-github-copilot/managing-your-github-copilot-subscription#canceling-your-github-copilot-subscription). To receive a refund you, please open a ticket on our [Support page](https://support.github.com/contact). If you are unable to access your account, our support team will be able to provide assistance

**Reply by ghost** [reply]
Thank you @iamcjwilliams , but as I said, my account has been deleted, hence I cannot visit the "Cancel your subscription" page.

**Comment by ghost** [comment, marked as answer]
The problem was on my side, I misread some messages, the account was not deleted. I opened a ticket and the support team was very helpful.
